_config.yml

```
@@ -12,11 +12,11 @@ description: >
   Student Robotics is an exciting annual competition challenging people aged
   16-18 to assemble a team and build fully autonomous robots.
 
-year: 2021
+year: 2022
 api_root: https://studentrobotics.org/comp-api
 
 competition_link: https://studentrobotics.org/events/sr2021/knockouts/
```

Let's also update this url
```suggestion
competition_link: https://studentrobotics.org/events/sr2022/competition/
```